accounting/defines.py

```diff
@@ -0,0 +1,31 @@
+DOCUMENT_TYPE_CHOICES = (
+        # Electronic check
+        ('', 'de_honorarios'),
+        ('', 'de_compraventa'),
+        ('', 'de_prestacion_de_servicios'),
+        ('', 'comprobante_de_pago_electronico'),
+
+        # libros_contables
+        ('', 'libros_de_contabilidad_principal'),
+        ('', 'libro_caja'),
+        ('', 'libro_diario'),
+        ('', 'libro_mayor'),
+        ('', 'libro_de_honorarios'),
+        ('', 'libro_de_inventarios_y_balances'),
+        ('', 'libros_de_control'),
+        ('', 'libro_fut'),
+        ('', 'libros_auxiliares'),
+        ('', 'libro_de_compras'),
+        ('', 'libro_de_remuneraciones'),
+        ('', 'libro_de_retenciones'),
+        
+        ('', 'notas_de_credito'),
+        ('', 'notas_de_debito'),
+        ('', 'letras_de_cambio'),
+        ('', 'pagares'),
```

Comment:
Es necesario agregar el primer valor de la tupla, de lo contrario quedara un string vacio en bd
```suggestion
        ('DH', 'de_honorarios'),
        ('DCV', 'de_compraventa'),
        ('DPS', 'de_prestacion_de_servicios'),
        ('CDPE', 'comprobante_de_pago_electronico'),

        # libros_contables
        ('LDCP', 'libros_de_contabilidad_principal'),
        ('LC', 'libro_caja'),
        ('LD', 'libro_diario'),
        ('LM', 'libro_mayor'),
        ('LDH', 'libro_de_honorarios'),
        ('LDIYB', 'libro_de_inventarios_y_balances'),
        ('LDC', 'libros_de_control'),
        ('LF', 'libro_fut'),
        ('LA', 'libros_auxiliares'),
        ('LDCOMP', 'libro_de_compras'),
        ('LDREM', 'libro_de_remuneraciones'),
        ('LDRET', 'libro_de_retenciones'),
        
        ('NDC', 'notas_de_credito'),
        ('NDB', 'notas_de_debito'),
        ('LDC', 'letras_de_cambio'),
        ('P', 'pagares'),
```